apps/camera/js/controllers/camera.js

```diff
@@ -97,6 +97,18 @@ CameraController.prototype.bindEvents = function() {
   debug('events bound');
 };
 
+CameraController.prototype.onRecordingChange = function(recording) {
+  var active;
+  if (recording === 'started') {
+    active = true;
+  } else if (recording === 'stopped') {
+    active = false;
+  } else {
+    return;
+  }
+  this.app.set('recording', active);
```

Comment:
Shouldn't we just set the `app` 'recording' property to the actual recording state instead of a boolean?